test/performance/odp_timer_accuracy.c

```diff
@@ -1268,6 +1302,12 @@ int main(int argc, char *argv[])
 		exit(EXIT_FAILURE);
 	}
 
+	argc = test_common_parse_options(argc, argv);
+	if (test_common_options(&common_options)) {
+		ODPH_ERR("Error: reading test helper options failed\n");
```

Comment:
To better follow the error message style in this application `Error:` could be dropped.